src/main/java/org/openrewrite/maven/ResourceParser.java

```diff
@@ -214,6 +220,11 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) {
             alreadyParsed.addAll(pythonPaths);
         }
 
+        if (!kotlinPaths.isEmpty()) {
+            sourceFiles = Stream.concat(sourceFiles, (Stream<S>) kotlinParser.parse(kotlinPaths, baseDir, ctx));
```

Comment:
This is a great start, but it requires a few changes.

The JavaParser in the ResourceParser is meant to parse Java sources that haven't been parsed in main or test.
These sources do not have the classpath setup and will be missing type attribution.

The main and test Java sources set the classpath like [this](https://github.com/openrewrite/rewrite-maven-plugin/blob/main/src/main/java/org/openrewrite/maven/MavenMojoProjectParser.java#L310).